engineapi/engineapi/routes/metatx.py

```diff
@@ -414,3 +431,32 @@ async def delete_requests(
         raise EngineHTTPException(status_code=500)
 
     return deleted_requests
+
+
+# @app.post("/requests/{request_id}/complete", tags=["requests"])
+@app.post("/requests/complete", tags=["requests"])
+async def complete_call_request_route(
+    tx_hash: str = Form(...),
```

Comment:
I think we should define a JSON body for the request.

And a JSON response type, as well.

Current setup is inconsistent with the rest of our API.

Reply:
Ok

Reply:
You didn't make this change.